src/spyglass.cr

```diff
@@ -1,5 +1,3 @@
-require "./img/frame"
-
-module Spyglass
-  VERSION = "0.1.0"
-end
+require "bottle"
+require "./lib/sod"
+require "./spyglass/**"
```

Comment:
I'd prefer explicit imports here, makes it easier to see what's included in the public API.

Reply:
Yeah I plan on adding them. Just makes it a bit easier for me in the beginning if I import a whole blob.